experimental/Schemes/elliptic_surface.jl

```diff
@@ -528,26 +548,26 @@ function relatively_minimal_model(E::EllipticSurface)
     set_attribute!(X0, :is_reduced=>true)
     set_attribute!(X0, :is_integral=>true)
   end
-  E.Y = Y0
-  E.blowups = projectionsY
+  Y.Y = Y0
+  Y.blowups = projectionsY
 
-  # We need to rewrap the last maps so that the domain is really E
+  # We need to rewrap the last maps so that the domain is really Y
   last_pr = pop!(projectionsY)
-  last_pr_wrap = CoveredSchemeMorphism(E, codomain(last_pr), covering_morphism(last_pr))
+  last_pr_wrap = CoveredSchemeMorphism(Y, codomain(last_pr), covering_morphism(last_pr))
   push!(projectionsY, last_pr_wrap)
-  E.ambient_blowups = projectionsX
+  Y.ambient_blowups = projectionsX
 
-  E.ambient_exceptionals = ambient_exceptionals
+  Y.ambient_exceptionals = ambient_exceptionals
   piY = CompositeCoveredSchemeMorphism(reverse(projectionsY))
-  E.blowup = piY
+  Y.blowup = piY
 
-  inc_Y0_wrap = CoveredClosedEmbedding(E, codomain(inc_Y0), covering_morphism(inc_Y0), check=false)
-  E.inc_Y = inc_Y0_wrap
+  inc_Y0_wrap = CoveredClosedEmbedding(Y, codomain(inc_Y0), covering_morphism(inc_Y0), check=false)
+  Y.inc_Y = inc_Y0_wrap
 
-  set_attribute!(E, :is_irreducible=> true)
-  set_attribute!(E, :is_reduced=>true)
-  set_attribute!(E, :is_integral=>true)
-  return E, piY
+  set_attribute!(Y, :is_irreducible=> true)
+  set_attribute!(Y, :is_reduced=>true)
+  set_attribute!(Y, :is_integral=>true)
```

Comment:
This must be `E`! Otherwise, the attribute is set only for the `underlying_scheme` and that will lead to massive tests and failures for further methods.

Reply:
I renamed E to Y. The final blown up surface is Y0